app/_hub/kong-inc/ai-proxy/_changelog.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### {{site.base_gateway}} 3.7.x
+
+* You can now enable [streaming on the AI Proxy plugin](/hub/kong-inc/ai-proxy/how-to/streaming/).
+* You can now [use an existing OpenAI-compatible SDK](/hub/kong-inc/ai-proxy/how-to/sdk-usage) (for example, Python OpenAI) to call different models, providers, and configurations with Kong AI Gateway.
```

Comment:
```suggestion
* You can now [use an existing OpenAI-compatible SDK](/hub/kong-inc/ai-proxy/how-to/sdk-usage) (for example, Python OpenAI) to call different models, providers, and configurations with Kong AI Gateway.
* **Enterprise feature:** You can now use Azure's native authentication mechanism to [secure your cloud-hosted models](/hub/kong-inc/ai-proxy/how-to/cloud-provider-authentication/).
```
This link will be broken for now, but it's here: https://docs.konghq.com/hub/kong-inc/ai-proxy/unreleased/how-to/cloud-provider-authentication/